### ISSUE TITLE
RockyLinux 9.1 cannot display the distribution logo

### ISSUE THREAD

**Issue by ghost** [dylanaraps/neofetch#2252]
## Description

RockyLinux 9.1 cannot display the distribution logo

#### Neofetch version

neofetch-7.1.0-7.el9.noarch

## Screenshot
![rocky](https://user-images.githubusercontent.com/111113097/205424968-55ed15ea-d2cd-4263-87a0-a31c756d1bd7.png)

## Config file

default

## Verbose log

[neofetchlog.txt](https://github.com/dylanaraps/neofetch/files/10145696/neofetchlog.txt)

**Comment by Qubik65536** [comment]
Hi, follow [these steps](https://github.com/dylanaraps/neofetch/wiki/Installation#latest-git-master-bleeding-edge) (install `neofetch` directly from git source) solves the problem for me. It seems that the `neofetch` installed from packaging manager / release doesn't include the support for Rocky Linux (8 & 9).

**Comment by iMonZ** [comment]
> Hi, follow [these steps](https://github.com/dylanaraps/neofetch/wiki/Installation#latest-git-master-bleeding-edge) (install `neofetch` directly from git source) solves the problem for me. It seems that the `neofetch` installed from packaging manager / release doesn't include the support for Rocky Linux (8 & 9).

How can this be fixed?

**Comment by Qubik65536** [comment]
> How can this be fixed?

@iMonZ I'm not sure what do you mean by "fixed". You can either install `neofetch` from source or use an up-to-date fork to get the logo for newer OSs. And from what I know, unless the dev of `neofetch` release a more recent version, there's no other way for us to install `neofetch` from a package manager and make it work.

------

P.S. If you really want to install the program with a package manager, [hykilpikonna/hyfetch](https://github.com/hykilpikonna/hyfetch) is the only one that I can find who works with it (it uses `pip` to install), I didn't find any other forks that can be installed from a package manager.

**Comment by iMonZ** [comment]
> > How can this be fixed?
> 
> @iMonZ I'm not sure what do you mean by "fixed". You can either install `neofetch` from source or use an up-to-date fork to get the logo for newer OSs. And from what I know, unless the dev of `neofetch` release a more recent version, there's no other way for us to install `neofetch` from a package manager and make it work.
> 
> P.S. If you really want to install the program with a package manager, [hykilpikonna/hyfetch](https://github.com/hykilpikonna/hyfetch) is the only one that I can find who works with it (it uses `pip` to install), I didn't find any other forks that can be installed from a package manager.

It can be fixed if the repo would update neofetch…

**Comment by Qubik65536** [comment]
> > > How can this be fixed?
> > 
> > 
> > @iMonZ I'm not sure what do you mean by "fixed". You can either install `neofetch` from source or use an up-to-date fork to get the logo for newer OSs. And from what I know, unless the dev of `neofetch` release a more recent version, there's no other way for us to install `neofetch` from a package manager and make it work.
> > P.S. If you really want to install the program with a package manager, [hykilpikonna/hyfetch](https://github.com/hykilpikonna/hyfetch) is the only one that I can find who works with it (it uses `pip` to install), I didn't find any other forks that can be installed from a package manager.
> 
> It can be fixed if the repo would update neofetch…

Yes... Or we can switch to an actively maintained fork... That's about all the solutions for us now.

**Comment by iMonZ** [comment]
> > > > How can this be fixed?
> > > 
> > > 
> > > @iMonZ I'm not sure what do you mean by "fixed". You can either install `neofetch` from source or use an up-to-date fork to get the logo for newer OSs. And from what I know, unless the dev of `neofetch` release a more recent version, there's no other way for us to install `neofetch` from a package manager and make it work.
> > > P.S. If you really want to install the program with a package manager, [hykilpikonna/hyfetch](https://github.com/hykilpikonna/hyfetch) is the only one that I can find who works with it (it uses `pip` to install), I didn't find any other forks that can be installed from a package manager.
> > 
> > 
> > It can be fixed if the repo would update neofetch…
> 
> Yes... Or we can switch to an actively maintained fork... That's about all the solutions for us now.

Which one?
